LSTM/TimeSeries-Univariate/src/LSTMTimeSeriesUnivariate.cpp

```diff
@@ -188,10 +189,10 @@ int main()
   dataset = dataset.submat(1, 1, 1, dataset.n_cols - 1);
 
   // Split the dataset into training and validation sets.
-  arma::mat trainData = dataset.submat(arma::span(),arma::span(0, (1 - RATIO) *
+  arma::mat trainData = dataset.submat(arma::span(), arma::span(0, (1 - RATIO) *
       dataset.n_cols));
-  arma::mat testData = dataset.submat(arma::span(), arma::span((1 - RATIO) * dataset.n_cols,
-      dataset.n_cols - 1));
+  arma::mat testData = dataset.submat(arma::span(),
+      arma::span((1 - RATIO) * dataset.n_cols, dataset.n_cols - 1));
```

Comment:
This and the similar changes are what I was thinking when I mention questionable wrapping decisions on my part

Reply:
I think it's an improvement.